doc/index.rst

```diff
@@ -1,25 +1,52 @@
-Welcome to yaml2sbml's documentation!
-===============================================
-
-This package allows the user to convert a system of ODEs specified in a yaml file to SBML. In addition, if experimental data is provided in the yaml file it can also be converted to PEtab.
-
-
-Installation
----------------------------
-
-`yaml2sbml` can be installed with `pip`:
-
-```pip install yaml2sbml```
-
+yaml2sbml
+=========
+
+.. image:: https://github.com/yaml2sbml-dev/yaml2sbml/workflows/CI/badge.svg
+   :target: https://github.com/yaml2sbml-dev/yaml2sbml/actions
+   :alt: Build status
+.. image:: https://codecov.io/gh/yaml2sbml-dev/yaml2sbml/branch/master/graph/badge.svg
+   :target: https://codecov.io/gh/yaml2sbml-dev/yaml2sbml
+   :alt: Code coverage
+.. image:: https://app.codacy.com/project/badge/Grade/dc25c9a84ba54710bbb23a6e08ab5d22
+   :target: https://www.codacy.com/manual/martamatos/yaml2sbml/dashboard?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=yaml2sbml-dev/yaml2sbml&amp;utm_campaign=Badge_Grade
```

Comment:
`martamatos/yaml2sbml/...`in the kink again :)

Reply:
I'm not sure if I have access to those codacy things. Could you update with the correct links?

Reply:
Is already done in the other README PR, we can skip for now :)